checks/branch_protection_test.go

```diff
@@ -174,7 +174,7 @@ func TestReleaseAndDevBranchProtected(t *testing.T) {
 				Error:         nil,
 				Score:         4,
 				NumberOfWarn:  9,
-				NumberOfInfo:  12,
+				NumberOfInfo:  11,
```

Comment:
this change is expected, just posting for my own record, the decrease is from codeowners being enabled, but no files. Previously this situation would produce 1 warn + 1 log. now it's just 1 warn.